### DISCUSSION TITLE
Add a new "cheat list" ListView like DuckStation already has

### DISCUSSION THREAD

**Discussion by ghost** [grumpycoders/pcsx-redux#1120] [Ideas]
DuckStation has a cheat manager with "cheat list" ListView that allows not advanced users to both quickly and easily use any cheat from CodeBreaker, GameShark, Action Replay and etc for the PS1 rom that they are playing.

I would like that PCSX-Redux also has this "cheat list" ListView like DuckStation has.

If not then at least add a new interface, like both ePSXe and PCSX Reloaded have, to use cheats from the internet but the user will have to do some work in order to use them on PCSX-Redux.

**Comment by wheremyfoodat** [comment]
This is a bit hard without cheat support

**Reply by ghost** [reply]
> This is a bit hard without cheat support

What do you mean exactly? And even if this is hard at least this is possible and feasible and that's what important.

**Reply by wheremyfoodat** [reply]
The emulator does not support cheats at the moment.

**Reply by ghost** [reply]
> The emulator does not support cheats at the moment.

And when the emulator will support cheats in the future?
